cmd/importVoucher.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright © 2025 NAME HERE <EMAIL ADDRESS>
+
+*/
+package cmd
```

Comment:
Make sure to have an empty line above package or the copyright stuff will end up as package docs!